### DISCUSSION TITLE
MountWizzard4 Wont Start

### DISCUSSION THREAD

**Discussion by jameshawks60** [mworion/MountWizzard4#351] [Q&A]
Hello All,

I have tried installing MW4 but it just wont start. Here is the error code that is returned from the log:

> [2024-03-30 17:55:09.342][E][    loggerMW.py][  46]          class MyApp(QApplication):
> [2024-03-30 17:55:11.045][H][      loader.py][ 296] ----------------------------------------------------------------------------------------------------
> [2024-03-30 17:55:11.061][H][      loader.py][ 297] mountwizzard4    : 3.2.5
> [2024-03-30 17:55:11.061][H][      loader.py][ 298] platform         : Windows
> [2024-03-30 17:55:11.061][H][      loader.py][ 299] sys.executable   : c:\MountWizard\venv\Scripts\python.exe
> [2024-03-30 17:55:11.061][H][      loader.py][ 300] actual workdir   : c:\MountWizard
> [2024-03-30 17:55:11.061][H][      loader.py][ 301] machine          : AMD64
> [2024-03-30 17:55:11.061][H][      loader.py][ 302] cpu              : Intel64 Family 6 Model 92 Stepping 9, GenuineIntel
> [2024-03-30 17:55:11.061][H][      loader.py][ 303] release          : 10
> [2024-03-30 17:55:11.061][H][      loader.py][ 304] python           : 3.9.0
> [2024-03-30 17:55:11.061][H][      loader.py][ 305] python runtime   : 64bit
> [2024-03-30 17:55:11.061][H][      loader.py][ 306] PyQt5 / Qt       : 5.15.9 / 5.15.2
> [2024-03-30 17:55:11.061][H][      loader.py][ 307] node / hostname  : ASTROPHOTO / ASTROPHOTO
> [2024-03-30 17:55:11.061][H][      loader.py][ 308] run as admin     : yes
> [2024-03-30 17:55:11.061][H][      loader.py][ 309] ----------------------------------------------------------------------------------------------------
> [2024-03-30 17:55:11.076][I][      loader.py][ 332] Using existing: [de440_mw4.bsp]
> [2024-03-30 17:55:11.092][I][      loader.py][ 332] Using existing: [visual.txt]
> [2024-03-30 17:55:11.092][I][      loader.py][ 332] Using existing: [CDFLeapSeconds.txt]
> [2024-03-30 17:55:11.107][I][      loader.py][ 332] Using existing: [tai-utc.dat]
> [2024-03-30 17:55:11.107][I][      loader.py][ 332] Using existing: [finals2000A.all]
> [2024-03-30 17:55:11.139][I][      loader.py][ 332] Using existing: [finals.data]
> [2024-03-30 17:55:11.170][I][      loader.py][ 332] Using existing: [mountwizzard4.pdf]
> [2024-03-30 17:55:11.201][I][     profile.py][ 189] Config file c:\MountWizard\config\config.cfg not existing
> [2024-03-30 17:55:11.357][I][     obsSite.py][ 142] Timescale is using downloaded version
> [2024-03-30 17:55:23.400][D][    startup.pyz][  77] Exit code:[3221225477], message:[], success:[False]

When I analyze the dmp file it has the following information:

> SYMBOL_NAME:  Qt5Core+1d69db
> MODULE_NAME: Qt5Core
> IMAGE_NAME:  Qt5Core.dll
> STACK_COMMAND:  ~17s; .ecxr ; kb
> FAILURE_BUCKET_ID:  INVALID_POINTER_READ_c0000005_Qt5Core.dll!Unknown
> OS_VERSION:  10.0.19041.1
> BUILDLAB_STR:  vb_release
> OSPLATFORM_TYPE:  x64
> OSNAME:  Windows 10
> IMAGE_VERSION:  5.15.2.0
> FAILURE_ID_HASH:  {e187daa5-db35-d98d-1840-c9f4590b5968}

With it being an invalid read in the Qt5Core.dll I am assuming the issue is with PyQt5. I have tried a fresh install of Python (various versions from 8.2 to 10) and of MW4 using v2 install and v3.

Any thoughts or help would be greatly appreciated.

**Comment by mworion** [comment]
Hi James,

sorry to hear about your troubles and thanks for analyzing the dump as well. I will have some time tomorrow to try to find a way around. As I cannot alter the Qt5 library, I have to find a workaround. Are you willing to make some test for that ? This would help as I do not see this issue in my environments.
BTW: Which network setup do you use ?
Michel

**Comment by mworion** [comment]
...and I made an Issue out of that: 
https://github.com/mworion/MountWizzard4/issues/352
If you don't mind we follow up in the issue thread.

**Comment by jameshawks60** [comment]
Michael,

No worries at all. I appreciate the time and help.


Thank You,
---
James Hawks

**Comment by jameshawks60** [comment]
Michael,

Yes, I can run some tests for you when needed.
I currently RDP into a Beeklink Windows 10 miniPC. The network is a Google
Nest Mesh.


Thank You,
---
James Hawks
